concourse/steps/scan_container_images.mako

```diff
@@ -99,6 +101,20 @@ results_above_threshold, results_below_threshold, license_report = protecode.uti
   image_reference_filter=filter_function,
   cvss_version = CVSSVersion('${protecode_scan.cvss_version().value}'),
 )
+results_above_threshold = protecode_results.above
```

Comment:
as discussed earlier: do not filter when writing to compliance-db. just collect everything - will filter later

Reply:
These filtered sub lists aren't taken for result insertion.
```python
results_above_threshold = protecode_results.above
results_below_threshold = protecode_results.below
license_report = protecode_results.licenseReport

protecode.scanning_util.insert_results(
    protecode_results=protecode_results,
    compliancedb_cfg_name=compliancedb_cfg_name,
)
```
Dropping those completely would require refactoring. 
I'm aware of that being not pretty at all, but it is by far the simplest solution.